Tests/Tests.csproj

```diff
@@ -18,9 +18,11 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="6.7.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Http" Version="2.2.2" />
```

Comment:
This file can be reverted as this package isn't used

Reply:
Removed